package.json

```diff
@@ -28,7 +28,7 @@
     "@envelop/parser-cache": "^2.2.0",
     "@envelop/validation-cache": "^2.2.0",
     "@faststore/api": "^1.6.7",
-    "@faststore/sdk": "^1.6.7",
+    "@faststore/sdk": "https://pkg.csb.dev/vtex/faststore/commit/4543b934/@faststore/sdk",
```

Comment:
Just a reminder to update this before merging :)

Reply:
Yeah, sure!

Reply:
@victorhmp fixed.